aperturedb/ImageLoader.py

```diff
@@ -86,11 +118,50 @@ def load_image(self, filename):
 
         return False, None
 
+    def load_url(self, url):
+
+        retries = 0
+        while True:
+            imgdata = requests.get(url)
+            if imgdata.ok:
+                if self.check_image:
+                    filename = "/tmp/" + url.split("/")[-1]
+                    fd = open(filename, "wb")
```

Comment:
you can avoid writing to a file and checking the image from buffer directly. Here is an example:
https://github.com/luisremis/visual_storm/blob/master/yfcc100m/python/eval/mysql_eval/MySQLQuery.py#L212